### PR TITLE
add how to install

### PR DESCRIPTION
**Pull request by ItoYo16u**
Hi, I added how to install section to README.md since I had a bit difficulty finding minimum requirements to use http4s.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,18 @@ val http = HttpRoutes.of {
     Ok("Hello, better world.")
 }
 ```
+## Install
+
+```scala
+for scala 2.x
```
Comment (ItoYo16u) [inline README.md:16]:
```suggestion
// for scala 2.x
```

File: README.md [hunk 2]
```diff
@@ -10,6 +10,18 @@ val http = HttpRoutes.of {
...
+val http4sVersionReleased = "0.21.24"
+for scala 3
```
Comment (ItoYo16u) [inline README.md:18]:
```suggestion
// for scala 3
```

**Comment by aeons** [comment]
This is all covered on the website: https://http4s.org/versions/

**Comment by rossabaker** [comment]
And starting with 0.22, it'll work for Scala 2 and Scala 3.  I guess it doesn't hurt to link to the versions page from this page, since it's confusing new users?

**Pull request closed without merging**